dropshot_endpoint/src/api_trait.rs

```diff
@@ -154,13 +157,61 @@ impl ApiMetadata {
     }
 }
 
+/// A mirror of dropshot's `TagConfig`, used as part of arguments to the
+/// top-level `api_description` macro.
+#[derive(Deserialize, Debug)]
+struct ApiTagConfig {
+    #[serde(default)]
+    allow_other_tags: bool,
+    #[serde(default)]
+    endpoint_tag_policy: ApiEndpointTagPolicy,
```

Comment:
similarly: do we want this to be `policy` ... ? I agree that it's nice to match TagConfig... but this seems potentially verbose

Reply:
Also done.